lib/virtio/virtio.c

```diff
@@ -79,3 +79,44 @@ void virtio_describe(struct virtio_device *dev, const char *msg,
 	// TODO: Not used currently - keeping it for future use
 	virtio_feature_name(0, desc);
 }
+
+int virtio_create_virtqueues(struct virtio_device *vdev, unsigned int flags,
+			     unsigned int nvqs, const char *names[],
+			     vq_callback *callbacks[])
+{
+	struct virtio_vring *rvring;
+	struct vring_alloc_info vring_info;
+	unsigned int num_vrings, i;
+	int ret;
+	(void)flags;
```

Comment:
if flags is void, lets remove it from the function as well.

Reply:
the flags here is for the features of the virtio user driver e.g. RPMsg, it just because we didn't implement that yet.
but maybe we should keep the argument in the API so that in future, when we we implement the feature, we don't need to change it.

Reply:
We should add a comment to the api that flags is for the future.